function-server/http-api.js

```diff
@@ -20,7 +21,16 @@ module.exports = (fun) => {
     app.use(/.*/, bodyParser.json({strict: false}));
 
     app.post(/.*/, async (req, res) => {
-        res.json(await fun(req.body));
+        let output = await fun(req.body)
+        if (output['type']) {
+            if (output['type'] === server.INPUT_ERROR) {
+                res.status = 400
+            }
+            if (output['type'] === server.FUNCTION_ERROR) {
+                res.status == 502
+            }
+        }
```

Comment:
This seems to have a similar issue to the previous Powershell one. If the user's function completes and returns a payload that contains `type` with `INPUT_ERROR` it will return `400` even if the function completed successfully.